### ISSUE TITLE
Thoughts on having a redux-bundler mono-repo

### ISSUE THREAD

**Issue by vujita** [HenrikJoreteg/redux-bundler#70]
@HenrikJoreteg I was wondering your thoughts of consolidating all the redux-bundle packages and bindings into one centralized repo possibly through something like [lerna](https://github.com/lerna/lerna)

Possibly even breaking out [extras like this](https://github.com/HenrikJoreteg/redux-bundler/tree/master/src/extras) into separate packages

Also any thoughts about making all these belong to a group.

Like having
"@redux-bundler/redux-bundler"
"@redux-bundler/redux-bundler-preact"
![image](https://user-images.githubusercontent.com/1197701/76154041-82892200-608a-11ea-9a41-789b12757d36.png)

**Comment by HenrikJoreteg** [comment]
It's something I've thought about, but obviously haven't done yet. Part of the reason I haven't is that Lerna specifically, feels a bit overwrought. I haven't really researched it. Are there other good options?

This looks kind of interesting... https://github.com/mateodelnorte/meta/blob/master/README.md

Keeps them separate, but let's you run commands across them. 

Personally I don't want to have import { X } from '@redux-bundler/redux-bundler' everywhere. The name just bugs me. Especially when I already have the top level name. That said, a little more unity would be nice.

**Comment by vujita** [comment]
Yarn [workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) offers a much lighter solution.  It does have a few draw backs, need to switch to yarn and possible issues with publishing that is listed

**Comment by davemo** [comment]
Alternatively I've heard good things about [Microsoft RushJS](https://rushjs.io), but haven't tried it out in a project yet

**Comment by odinho** [comment]
PouchDB does monorepo in a no-tooling way after trying Lerna: [Why we dropped Lerna from PouchDB](https://gist.github.com/nolanlawson/457cdb309c9ec5b39f0d420266a9faa4).

They're still using their simple ["Alle" monorepo method](https://github.com/boennemann/alle) these years later: https://github.com/pouchdb/pouchdb (I don't have direct experience with it myself, just remembered I'd like to try monorepo this way :) ).

**Comment by HenrikJoreteg** [comment]
Yeah, conceptually, I really like this idea. But I have yet to find a monorepo approach that doesn't feel belabored and frustrating. I'm open to trying it though. Just need to find the time :)
